server/routes/api/place.py

```diff
@@ -15,10 +15,11 @@
 import collections
 import json
 
-from flask import Blueprint, request
+from flask import Blueprint, request, jsonify
 
 from cache import cache
 from services.datacommons import fetch_data
+import services.datacommons as dc
```

Comment:
Please group import's together. This should move up to line 17

Reply:
Moved